src/pypgstac/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "pgstacrs"
+version = "0.7.10-dev"
```

```suggestion
version = "0.0.0"
```

Pre-releases aren't really supported: https://github.com/rust-lang/cargo/issues/2222. Also, I don't think we'll be releasing this crate on its own -- it's only meant for use in the Python bindings. So `0.0.0` will indicate that it's not meant for release to crates.io.